### PR TITLE
7469: fixed float heading

### PR DESCRIPTION
**Pull request by jpdimond**
woo!

image

### CODE REVIEW

**Comment by jpdimond** [comment]
I removed the float left on h1 in main, and tested in RTL and LTR, and all looks good!

File: app/assets/stylesheets/all/components/_forms.css.scss [hunk 1]
```diff
@@ -6,4 +6,17 @@ form.form_form {
   .sms-fields, .form_recipients {
     display: none; // Initially hidden
   }
-}
\ No newline at end of file
+}
+
+@media print {
+  /* The outer title on forms pages should be hidden since we include it separately. */
```
[the remark this answers is not in the document]
Comment (hooverlunch) [reply app/assets/stylesheets/all/components/_forms.css.scss]: Please change this to a `//` style comment.

File: app/assets/stylesheets/all/components/_forms.css.scss [hunk 2]
```diff
@@ -6,4 +6,17 @@ form.form_form {
...
+  .forms_index, .forms_show, .forms_edit {
+    h1.title {
+      display: none;
```
[the remark this answers is not in the document]
Comment (hooverlunch) [reply app/assets/stylesheets/all/components/_forms.css.scss]: Please add an empty line after the h1 style block.

File: app/assets/stylesheets/all/components/_forms.css.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (jpdimond) [reply app/assets/stylesheets/all/components/_forms.css.scss]: done

**Pull request merged**